public/app/plugins/datasource/azuremonitor/components/ConfigEditor/MonitorConfig.tsx

```diff
@@ -48,12 +48,15 @@ export const MonitorConfig = (props: Props) => {
     }
   });
 
+  // feature flag to enable/disable user auth only for Azure Monitor
+  const azureMonitorAuthEnabled = config.featureToggles.azureMonitorEnableUserAuth ?? true;
```

Comment:
Spotted a minor bug here (that is also a TIL). If a feature toggle is set to `false` then it's not passed to the frontend.  I think my suggestion below should handle this better 😊